### DISCUSSION TITLE
Security Onion & Cloudflare Tunnels

### DISCUSSION THREAD

**Discussion by mustarddehu** [Security-Onion-Solutions/securityonion#11424] [Unsupported Versions]
Running the latest version of security onion.

We would like to setup access to the security onion web interface via Cloudflare tunnels. We have added the fqdn we would like to use into the nginx config but when we hit the cloudflare URL the security onion login service that looks to run on port 9822 is rewriting the URL to the internal fqdn used by security onion that isnt accesible via cloudflare. 

I'm modifying the config inside `/opt/so/saltstack/default/salt/nginx/etc` is there another location I need to make a change?

**Comment by dougburks** [comment]
You shouldn't modify anything in /opt/so/saltstack/default/.

If you're running 2.4 and are trying to change the web access URL, please see:
https://docs.securityonion.net/en/2.4/url-base.html
